src/repr/src/row.proto

```diff
@@ -106,13 +106,8 @@ message ProtoTimestamp {
 message ProtoInterval {
     // A possibly negative number of months.
     int32 months = 1;
-    // A 128-bit nanosecond count, represented as a 64-bit lo and hi.
-    //
-    // TODO: Why do our in-mem intervals use this representation? I've seen
-    // month/day/nanos in the past becuase a day is not always the same number
-    // of nanos.
-    int64 duration_lo = 2;
-    int64 duration_hi = 3;
+    int32 days = 2;
```

Comment:
@danhhz Is this OK? Are our persisted values baked in enough yet that we need to write migrations?

Reply:
yup we're still good! we do have the background system tables test, but all that's needed to avoid breaking that is to bump mz_persist::gen::ProtoMeta::ENCODING_VERSION up by 1 and then fix up the resulting unit test breakage (golden_test at least, I don't recall offhand if there are others).

Reply:
Yeah, that TODO is just saying the cleanup _can_ be done now, not that it has to be done on any particular sort of timeline. Definitely go ahead and leave that to us :)